### ISSUE TITLE
text formatting style for lectures

### ISSUE THREAD

**Issue by ozanj** [Rucla-ed/rclass2#4]
would be good to develop some guidelines for how lectures are formatted.

in particular, when writing lecture .Rmd files , I am unsure when to use __bold__, when to use _italics_ or when to use `code` text style.

also unsure when to use level-1 -2 or -3 headings.

any good rules of thumb that you all know of from other courses you have taken? from an internet resource?

**Comment by mpatricia01** [comment]
In Christensen's class he used boldface for headings and to emphasize something. In my class last quarter the professor used a different color text (blue or red) to emphasize something. 

I think consistency matters most. We could bold headings and text we want to emphasize. We could include `code` when we are referring to functions or packages etc.

**Comment by cyouh95** [comment]
I think I've also seen textbooks where **bold** is used for key terms and _italics_ for emphasizing a main point/sentence. But whatever is used, I agree that consistency is probably most important!

For the headings, perhaps level-1 should cover the main topics (ie. if someone were to look at just the level-1 headings in a table of contents, they should know what kind of info the lecture contains). Division to level-2 or 3 heading might depend on each main section - but probably a good idea to start breaking things up into subsections if it is becoming too long.

**Comment by ozanj** [comment]
should we write some sort of document where we establish style guidelines for writing lectures?

**Comment by mpatricia01** [comment]
Yeah, we can. We can create some simple style guidelines. You are working on the first few lectures. Do you want to jot down some style guidelines you started (maybe in a google doc or r markdown file somewhere)  and then we can edit the document for lectures we take lead on?

**Comment by ozanj** [comment]
I started the following document

lectures >> _style >> style_guidelines.Rmd

just pushed initial changes. document contains a lot of questions where I am not sure what best practice is. you all should revise/add to this document.

**Comment by ozanj** [comment]
@mpatricia01 @cyouh95 

why don't you two meet at some point to discuss/edit the styles_guidelines doc that I created. and once you meet, close out this issue.
